sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md

```diff
@@ -0,0 +1,107 @@
+# Asynchronously list Key Vault secrets with Azure Core Tracing OpenTelemetry
+ 
+Following documentation describes instructions to run a sample program for asynchronously creating and listing secrets of a Key Vault with tracing instrumentation.
+
+## Getting Started
+Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package and **[Logging Exporter][logging_exporter]** as exporter.
+### Adding dependencies to your project:
+```xml
+<dependencies>
+    <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-sdk</artifactId>
+        <version>0.2.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-security-keyvault-secrets</artifactId>
+        <version>4.0.0</version>
+    </dependency>
+    <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-core-tracing-opentelemetry</artifactId>
+        <version>1.0.0-beta.1</version>
+    </dependency>
+    <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-exporters-logging</artifactId>
+        <version>0.2.0-SNAPSHOT</version>
+    </dependency>
+</dependencies>
+```
+
+#### Sample demonstrates tracing when asynchronously creating and listing secrets from a Key Vault using [azure-security-keyvault-secrets][azure_keyvault_secrets] client library.
+```java
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.security.keyvault.secrets.SecretAsyncClient;
+import com.azure.security.keyvault.secrets.SecretClientBuilder;
+import com.azure.security.keyvault.secrets.models.KeyVaultSecret;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+import reactor.util.context.Context;
+
+import java.util.logging.Logger;
+
+import static com.azure.core.util.tracing.Tracer.PARENT_SPAN_KEY;
+import static java.util.logging.Logger.getLogger;
+
+/**
+ * Sample demonstrates tracing how to add and list secrets in a Key vault with tracing enabled with a Logging Exporter.
+ */
+public class Sample {
+
+    private static final Logger LOGGER = getLogger("Sample");
+    private static final Tracer TRACER;
+    private static final TracerSdkFactory TRACER_SDK_FACTORY;
+
+    static {
+        TRACER_SDK_FACTORY = Helper.configureOpenTelemetryAndJaegerExporter(LOGGER);
+        TRACER = TRACER_SDK_FACTORY.get("Azure-OpenTelemetry");
+    }
+
+    public static void main(String[] args) throws InterruptedException {
+        Span userSpan = TRACER.spanBuilder("user-parent-span").startSpan();
+        final Scope scope = TRACER.withSpan(userSpan);
+        LOGGER.info("=== Start user scoped span  ===");
```

Comment:
logging all calls to opentelemetry seems to defeat purpose of tracing - what do you want to demonstrate with it? Can we do it with stdout exporter, switch to comments or just avoid it completely?

Reply:
I can remove the logging comments, I earlier had them for Jaegar exporter.